meinberlin/apps/kiezradar/models.py

```diff
@@ -0,0 +1,64 @@
+from django.conf import settings
+from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from adhocracy4.maps.fields import PointField
+
+
+class KiezRadarFilter(models.Model):
+    topic = models.CharField(max_length=126, unique=True)
```

Comment:
I'm not entirely sure if I follow the structure here, do we need the `KiezRadarFilter` because it will be extended at some point in the future? Because if it only has the `topic` we could also just have a m2m to the `Topic` model on the `SearchProfile`, right? And in the case here: would it make more sense for the topic to be a foreign key on `Topic` instead of a `CharField` ? This way only topics which are defined can be added

Reply:
I understood the multiselect filter later will have options that are added either by admins/initiators, or on the fly like keywords, while creating the search profile, not from the topics which come from the django settings. Also the topic field here could be a json field which combines different topics. Need to ask PM for clarification.

Reply:
my understanding of the story was that currently we only add the existing filters:
- project title (free text)
- organisation (free text)
- type (informational, participation, ...)
- status (all, running, ended) 
- district
- topic (as in the settings)

and you can select multiple options per filter (except for status filter). Maybe we can talk about it in daily again with the responsible PM as I don't remember anything about admins/initiators being able to add options so might be good to clarifiy

Reply:
Clarified now, topics are the one predefined. Bur multiple filters for all categories, will result in search profiles that are almost similar for different free text searches, and too many relations.